packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -208,7 +358,11 @@ it('should validate the screenshot with options', () => {
 
 ### Terra.validates.element
 
-The element assertion basically performs both `Terra.validates.accessibility` and `Terra.validates.screenshot` assertions. Like these two assertions the element assertion should also only be invoked inside an `it` block.
+The validates element assertion performs both accessibility and screenshot assertions.
+
+<Notice variant="caution" ariaLevel="3">
+  The validates element assertion must be used within an `it` block.
```

Comment:
Do we want to also include `and a screenshot name must be provided.` to be consistent with what you added in validates.screenshot?

Reply:
```suggestion
  The validates element assertion must be used within an `it` block and a screenshot name must be provided.
```

Reply:
Updated:

https://github.com/cerner/terra-toolkit/pull/610/commits/36606851682a8a89e5cc76e7a6ec39436bd45d0c